presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,452 @@ Except for the limit on queued queries, when a resource group runs out of a reso
 
+File Resource Group Manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: resources-groups-example.json
+    :language: json
+
+Database Resource Group Manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This example is for a MySQL database.
+
+.. literalinclude:: resource-groups-example1.sql
+    :language: sql
+
+.. code-block:: sql
+
+    -- global properties
+
+    -- get ID of 'other' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'other';  -- 4
+
+    -- create '${USER}' group with 'other' as parent.
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, environment, parent) VALUES ('${USER}', '10%', 1, 100, 'test_environment', 4);
+
+    -- create 'bi-${toolname}' group with 'adhoc' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, scheduling_policy, environment, parent) VALUES ('bi-${toolname}', '10%', 10, 100, 10, 'weighted_fair', 'test_environment', 3);
+
+    -- create 'pipeline' group with 'global' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, jmx_export, environment, parent) VALUES ('pipeline', '80%', 45, 100, 1, true, 'test_environment', 1);
+
+    -- create a root group 'global' with NULL parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_policy, jmx_export, environment) VALUES ('global', '80%', 100, 1000, 'weighted', true, 'test_environment');
+
+    -- get ID of 'global' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'global';  -- 1
+    -- create two new groups with 'global' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, environment, parent) VALUES ('data_definition', '10%', 5, 100, 1, 'test_environment', 1);
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, environment, parent) VALUES ('adhoc', '10%', 50, 1, 10, 'test_environment', 1);
+
+    -- get ID of 'adhoc' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'adhoc';   -- 3
+
+    -- create 'other' group with 'adhoc' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, scheduling_policy, environment, parent) VALUES ('other', '10%', 2, 1, 10, 'weighted_fair', 'test_environment', 3);
+
+--In below examples, three selectors are created for user1, user2, and user3 respectively.
+The resource_group_id is retrieved using a SELECT statement from the resource_groups table
+where the name matches the user name. The user_regex is set to match the respective user name and
+the priority is set to 1, 2, and 3 respectively.
+
+Please note that you would need to have the respective resource groups (i.e., user1, user2, user3)
```

Comment:
```suggestion
Please note that you would need to have the respective resource groups (that is, user1, user2, user3)
```

It is recommended to not use Latin abbreviations, such as i.e., e.g., or &. 
- For i.e, use that is. 
- For e.g., it is recommended to use for example, such as, for instance, or like instead. 
- For &, use and instead, unless you are documenting a UI element that uses an &.

Above advice taken from the [GitLab docs Recommended Word list](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#ie).

If you would, please make this change throughout the file.